spec/directives/extend/error.hrx

```diff
@@ -0,0 +1,55 @@
+<===> complex/input.scss
+a b {
+  a: b;
+}
+c {
+  @extend a b;
+}
+
+<===> complex/error
+Error: complex selectors may not be extended.
+  ,
+5 |   @extend a b;
+  |           ^^^
+  '
+  input.scss 5:11  root stylesheet
+
+<===> complex/error-libsass
+Error: complex selectors may not be extended.
+        on line 5:11 of input.scss
+>>   @extend a b;
+
+   ----------^
+
+<===>
+================================================================================
+<===> compound/input.scss
+a:hover {
+  a: b;
+}
+b {
+  @extend a:hover;
+}
+
+<===> compound/output-libsass.css
```

Comment:
I probably makes sense to mark this one as TODO for libsass, since the correct behavior is now an error, not a warning.

Reply:
Well, we have lots of other cases in sass-spec that have a libsass alternative showing a deprecation warning instead of triggering an error (that one misses the `DEPRECATION` part though)

Reply:
@jathak but will it stay deprecated ? @mgreter is working on the 4.0 version of libsass in https://github.com/sass/libsass/pull/3135